git-config/src/values.rs

```diff
@@ -250,6 +257,7 @@ impl Display for Value<'_> {
             Value::Boolean(b) => b.fmt(f),
             Value::Integer(i) => i.fmt(f),
             Value::Color(c) => c.fmt(f),
+            Value::Path(p) => p.value.to_str().expect("path is valid utf-8").fmt(f),
```

Comment:
Depending on how the path's encoding is handled, I think it would be very similar if not the same as `Value::Other()`.

Reply:
May be, but then there are not std::path::Path related methods, not sure of these are needed. So good reason to look at git